### PR TITLE
Online meetings frontend styling

### PR DESCRIPTION
**Pull request by tech2serve**
This adds front end, widget styling & single meeting page styling for online meetings. 

To help reduce the clutter next to the meeting name, the temporary closure flag is not displayed if the meeting is also online; instead only "Online meeting" is displayed. The server side logic for this could be cleaned up, but it seems to work so in the interest of speed. We can refactor later.

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -423,18 +423,15 @@ function tsml_format_name($name, $types=null) {
 	if (!is_array($types)) $types = array();
 	if (empty($tsml_programs[$tsml_program]['flags']) || !is_array($tsml_programs[$tsml_program]['flags'])) return $name;
 	$append = array();
-	$is_meeting_online = in_array("ONL", $types, TRUE);
+	$meeting_is_online = in_array("ONL", $types, TRUE);
```
Comment (joshreisner) [inline includes/functions.php]: better to use single quotes

Comment (joshreisner) [reply includes/functions.php]: also the `, TRUE` wouldn't do anything in this case

File: assets/src/public.js [hunk 2]
```diff
@@ -521,13 +521,16 @@ jQuery(function($){
...
-					let isMeetingTempClosedAndOnline = (obj.types.indexOf('ONL') !== -1) && (obj.types.indexOf('TC') !== -1)
-					console.log(obj.name, isMeetingTempClosedAndOnline)
 					//add type 'flags'
 					if (typeof tsml.flags == 'object') {
+						// True if the meeting is temporarily closed, but online option available
+						var meetingIsOnlineAndTC = (obj.types.indexOf('TC') !== -1) && (obj.types.indexOf('ONL') !== -1)
 						for (var i = 0; i < tsml.flags.length; i++) {
-
-							if ((obj.types.indexOf(tsml.flags[i]) !== -1) && (isMeetingTempClosedAndOnline && tsml.flags[i] !== 'TC')) {
+							flagIsTempClosed = tsml.flags[i] === 'TC'
```
Comment (joshreisner) [inline assets/src/public.js]: prob need a `var` here for consistency

Comment (tech2serve) [reply assets/src/public.js]: I agree! Not sure why that didn't error?

File: includes/functions.php (hunk 1 above)
Comment (tech2serve) [reply includes/functions.php]: Both changed

**Pull request merged**